tsl/src/continuous_aggs/common.c

```diff
@@ -1569,3 +1569,31 @@ time_bucket_info_has_fixed_width(const CAggTimebucketInfo *tbinfo)
 		return tbinfo->bucket_time_width->month == 0 && tbinfo->bucket_time_timezone == NULL;
 	}
 }
+
+ContinuousAgg *
+get_cagg_by_relid_or_fail(const Oid cagg_relid)
```

Comment:
```suggestion
cagg_get_by_relid_or_fail(const Oid cagg_relid)
```
Just to follow same pattern from other parts (i.e.: ts_chunk_get_by_relid)... to we need to prefix it with `ts_`?

Reply:
Good point, renamed